rosidl_runtime_py/rosidl_runtime_py/utilities.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Text
+
+from rosidl_parser.definition import NamespacedType
+
+
+def get_namespaced_type(path: Text):
+    """Create a `NamespacedType` object from the full name of an interface."""
+    return __get_namespaced_type(path)
+
+
+def get_action_namespaced_type(path: Text):
+    """Create a `NamespacedType` object from the full name of an action."""
+    return __get_namespaced_type(path, 'action')
+
+
+def get_message_namespaced_type(path: Text):
+    """Create a `NamespacedType` object from the full name of a message."""
+    return __get_namespaced_type(path, 'msg')
+
+
+def get_service_namespaced_type(path: Text):
+    """Create a `NamespacedType` object from the full name of a service."""
+    return __get_namespaced_type(path, 'srv')
```

Comment:
Nitpick: I would use the following order bases on increasingly higher concepts:
* `get_message_namespaced_type`
* `get_service_namespaced_type`
* `get_action_namespaced_type`

Same for the `is_*` functions below.